sdcm/report_templates/results_latency_during_ops_short.html

```diff
@@ -163,6 +166,19 @@ <h2>{{ operation }}</h2>
                             {% endfor %}
                         {% endif %}
                     </table>
+                    {% for cycle in results['cycles'] %}
```

Comment:
nitpick: i would add all of this new for-loop after the current line 182 `<span STYLE="font-size:12px" class="red">* All latency values are in ms. if latency has color red, check detailed HDR report</span>` so we have that little legend on the tables closer to the table

Reply:
Done